### PR TITLE
MAGECLOUD-4894: [MTS] Add support of split DB on docker

### PR DESCRIPTION
**Pull request by arhiopterecs**
## Purpose of this pull request
Adds instructions on how to prepare `magento-cloud-docker` (ece-docker) for deploying Magento in Split Database mode

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/cloud/docker/docker-split-db.md [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+```
+NOTES: As with the 'db' container, you can use the options to set the ports to forward to your local environment:
+  
+```shell script
+php vendor/bin/ece-docker build:compose --expose-db-quote-port=<port for quotee db service> --expose-db-sales-port=<port for sales db service>
+```
+
+6. Аfter generation, open the file `./docker-compose.yml` and you will see that the configuration for database services has been added:
+
+```shell script
```
Comment (meker12) [inline src/cloud/docker/docker-split-db.md:50]:
```suggestion
```bash
```

File: src/cloud/docker/docker-split-db.md [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+
+7. Run 
+
+```shell script
+docker-compose up -d
+```
+
+8. Run command 
+
+```shell script
```
Comment (meker12) [inline src/cloud/docker/docker-split-db.md:135]:
```suggestion
```bash
```

File: src/cloud/docker/docker-split-db.md [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+                'username' => 'magento2',
+                'port' => '3306'
+            ]
+        ],
+    ])),
+   //...
+];
+```
+
+7. Run
```
Comment (meker12) [inline src/cloud/docker/docker-split-db.md:127]:
```suggestion
7.  Using the updated configuration, build files to containers and run in the background.
```

File: src/cloud/docker/docker-split-db.md [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+            ]
+        ],
+    ])),
+   //...
+];
+```
+
+7. Run 
+
+```shell script
```
Comment (meker12) [inline src/cloud/docker/docker-split-db.md:129]:
```suggestion
```bash
```

File: src/cloud/docker/docker-split-db.md [hunk 5]
```diff
@@ -0,0 +1,187 @@
...
+docker-compose up -d
+```
+
+8. Run command
```
Comment (meker12) [inline src/cloud/docker/docker-split-db.md:133]:
```suggestion
8. Install Magento in your Docker environment.
```

File: src/cloud/docker/docker-split-db.md [hunk 6]
```diff
@@ -0,0 +1,187 @@
...
+      - 'mariadb-conf:/etc/mysql/mariadb.conf.d'
+      - 'docker-mnt:/mnt:delegated'
+    networks:
+      magento:
+        aliases:
+          - db-sales.magento2.docker
+...
+```
+     
+6. The file `./.docker/config.php.dist` was generated with next configurations:
```
Comment (meker12) [inline src/cloud/docker/docker-split-db.md:96]:
```suggestion
6.  Verify that the `./.docker/config.php.dist`  environment configuration file includes the updated database service configuration:
```

Comment (meker12) [reply src/cloud/docker/docker-split-db.md:96]: Should the filename be  `./.docker/config.php`

File: src/cloud/docker/docker-split-db.md [hunk 7]
```diff
@@ -0,0 +1,187 @@
...
+magento-cloud_db_1              docker-entrypoint.sh mysqld      Up             0.0.0.0:32872->3306/tcp
+...
+```
+
+12. Yuo can use this ports to connect to DB services from your host
+ 
+Now everything is ready to install Magento with Split Databases
+
+
+   ![Xdebug Helper options]({{ site.baseurl }}/common/images/cloud-xdebug_helper-options.png){:width="400px"}
```
Comment (meker12) [inline src/cloud/docker/docker-split-db.md:184]: Not sure why this image is here.  Maybe you can remove it. Or if you need it for something, add a sentence to explain why it is here.

File: src/cloud/docker/docker-split-db.md [hunk 8]
```diff
@@ -0,0 +1,187 @@
...
+
+[docker-config]: {{site.baseurl}}/cloud/docker/docker-config.html
+[Xdebug Helper extension]: https://chrome.google.com/webstore/detail/xdebug-helper/eadndfjplgieldjbigjakmdgkmoaaaoc?hl=en
```
Comment (meker12) [inline src/cloud/docker/docker-split-db.md:187]: Do you need the Xdebug Helper extension link?

File: src/cloud/docker/docker-split-db.md [hunk 9]
```diff
@@ -0,0 +1,187 @@
...
+```
+
+12. Yuo can use this ports to connect to DB services from your host
+ 
+Now everything is ready to install Magento with Split Databases
+
+
+   ![Xdebug Helper options]({{ site.baseurl }}/common/images/cloud-xdebug_helper-options.png){:width="400px"}
+
+[docker-config]: {{site.baseurl}}/cloud/docker/docker-config.html
```
Comment (meker12) [inline src/cloud/docker/docker-split-db.md:186]: This link is not referenced in the topic. You can remove, or if you need it add the reference to the topic text where it's needed.

File: src/cloud/docker/docker-split-db.md [hunk 10]
```diff
@@ -0,0 +1,187 @@
...
+```shell script
+docker-compose up -d
+```
+
+8. Run command 
+
+```shell script
+docker-compose run deploy bash
+```
+
```
Comment (meker12) [inline src/cloud/docker/docker-split-db.md:138]: I don't think that you need the rest of the steps in this topic.

Instead, after step 8 have an ending sentence that says something like: 

After deploying Magento with the split DB configuration, use the database credentials and port information to manage each database. See [Manage the database]({{site.baseurl}}/cloud/docker/docker-manage-database.html.

On the Manage database topic, you could update the sections "Connect to a database" and  "to connect to the database port" to include the split db example.

File: src/cloud/docker/docker-split-db.md [hunk 11]
```diff
@@ -0,0 +1,187 @@
+---
+group: cloud-guide
+title: Configure Split Databases for Docker
```
Comment (meker12) [inline src/cloud/docker/docker-split-db.md:3]:
```suggestion
title: Configure Magento split databases for Docker
```

File: src/cloud/docker/docker-split-db.md [hunk 12]
```diff
@@ -0,0 +1,187 @@
...
+
+```yaml
+relationships:
+    ...
+    database-quote: "mysql-quote:mysql"
+    database-sales: "mysql-sales:mysql"
+   ...
+```
+
+5. Let's now generate the file by running the cli command:
```
[the remark this answers is not in the document]
Comment (arhiopterecs) [reply src/cloud/docker/docker-split-db.md:38]:
```suggestion
5. Generate the Docker Compose configuration file for
```
That should be enough

File: src/cloud/docker/docker-split-db.md (hunk 5 above)
Comment (arhiopterecs) [reply src/cloud/docker/docker-split-db.md:133]: It is not an installation command. I will fix with next commit

File: src/cloud/docker/docker-split-db.md (hunk 7 above)
Comment (arhiopterecs) [reply src/cloud/docker/docker-split-db.md:184]: Fixed

File: src/cloud/docker/docker-split-db.md (hunk 9 above)
Comment (arhiopterecs) [reply src/cloud/docker/docker-split-db.md:186]: fixed

File: src/cloud/docker/docker-split-db.md [hunk 13]
```diff
@@ -0,0 +1,187 @@
...
+```bash
+php vendor/bin/ece-docker build:compose
+```
+NOTES: As with the 'db' container, you can use the options to set the ports to forward to your local environment:
+  
+```shell script
```
Comment (meker12) [inline src/cloud/docker/docker-split-db.md:44]:
```suggestion
```bash
```

File: src/cloud/docker/docker-split-db.md (hunk 10 above)
Comment (arhiopterecs) [reply src/cloud/docker/docker-split-db.md:151]: Done

**Pull request merged**